### PR TITLE
buildbot-postsubmit-job

### PR DESCRIPTION
**Pull request by RagniGarg97**
Prow job to build build-bot docker image: https://github.com/ppc64le-cloud/build-bot/blob/master/Dockerfile
1. build and push with --build-arg ARCH=amd64
2. build and push with --build-arg ARCH=ppc64le
3. run manifest create

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/107#" title="Author self-approved">RagniGarg97</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@RagniGarg97: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-test-infra-yamllint | 15521e4864251493443348c5c129b2780512da45 | [link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_test-infra/107/pull-test-infra-yamllint/1352531252833423360) | `/test pull-test-infra-yamllint`

[Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=test-infra&pr=107). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:RagniGarg97).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml:28]: Lets use `go get` instead of these lines

File: config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml:74]: Lets use the docker config

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by ltccci** [comment]
@RagniGarg97: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-test-infra-yamllint | 1a510db5510b6a8b773b66bbe86825bf3df89011 | [link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_test-infra/107/pull-test-infra-yamllint/1354567992964288512) | `/test pull-test-infra-yamllint`

[Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=test-infra&pr=107). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:RagniGarg97).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml:81]:
```suggestion
                manifest-tool --debug --docker-cfg `pwd` push from-spec registry.yaml
```

Lets not hardcode the path full path here!

**Review by mkumatag — CHANGES_REQUESTED** [review]
minor comment.

**Comment by mkumatag** [comment]
and fix the commit user as well

**Comment by RagniGarg97** [comment]
> and fix the commit user as well

fixed

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/107#issuecomment-768829770" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/107#" title="Author self-approved">RagniGarg97</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@RagniGarg97: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `buildbot-postsubmit.yaml` using file `config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/107):

> Prow job to build build-bot docker image: https://github.com/ppc64le-cloud/build-bot/blob/master/Dockerfile
>1. build and push with --build-arg ARCH=amd64
>2. build and push with --build-arg ARCH=ppc64le
>3. run manifest create


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
